configs/dfc-fermata.nrel-op.json

```diff
@@ -69,17 +69,17 @@
         {
           "surveyName": "DfcGasTrip",
           "not-filled-in-label": { "en": "Gas Car Survey" },
-          "showsIf": "sections[0]['sensed_mode_str'] == 'CAR'"
+          "showsIf": "confirmedMode?.baseMode == 'CAR'"
```

Comment:
I am fine with this for now but note, as a future fix, that this needs to be adjusted for python as well (@Abby-Wheelis)

Reply:
Yes. I have some ideas for that, but I want to use `e-mission-common` for it. I will discuss it with Abby once the rush of this "alpha" release has died down